website/docs/docs/deploy/merge-jobs.md

```diff
@@ -5,35 +5,61 @@ description: "Learn how to trigger a dbt job run when a Git pull request merges.
    - If you're using a native [GitLab](/docs/cloud/git/connect-gitlab) integration, you need a paid or self-hosted account that includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). If you're using GitLab Free, merge requests will trigger CI jobs but CI job status updates (success or failure of the job) will not be reported back to GitLab.
-- You are set up to receive push notifications from your Git provider. For details, refer to [Job notifications](/docs/deploy/job-notifications). 
+- For deferral (which is the default), make sure there has been at least one successful job run in the deferred environment.
 
 ## Set up job trigger on Git merge {#set-up-merge-jobs}
 
-1. On your deployment environment page, click **Create Job** > **Merge Job**. 
+1. On your deployment environment page, click **Create job** > **Merge job**. 
 1. Options in the **Job settings** section:
     - **Job name** &mdash; Specify the name for the merge job.
     - **Description** &mdash; Provide a descripion about the job. 
     - **Environment** &mdash; By default, it’s set to the environment you created the job from.
-1. In the **Git trigger** section, the **Run on merge** option is enabled by default. Every time a PR merges (to the main branch) in your Git repo, this job will get triggered to run. 
+1. In the **Git trigger** section, the **Run on merge** option is enabled by default. Every time a PR merges (to a base
+branch configured in the environment) in your Git repo, this job will get triggered to run. 
 1. Options in the **Execution settings** section:
     - **Commands** &mdash; By default, it includes the `dbt build --select state:modified+` command. This informs dbt Cloud to build only new or changed models and their downstream dependents. Importantly, state comparison can only happen when there is a deferred environment selected to compare state to. Click **Add command** to add more [commands](/docs/deploy/job-commands) that you want to be invoked when this job runs.
-    - **Compare changes against an environment (Deferral)** &mdash; By default, it's set to the environment you created the job from. This option allows dbt Cloud to check the state of the code in the PR against the code running in the deferred environment, so as to only check the modified code, instead of building the full table or the entire DAG.
+    - **Compare changes against** &mdash; By default, it's set to compare changes against the environment you created the job from. This option allows dbt Cloud to check the state of the code in the PR against the code running in the deferred environment, so as to only check the modified code, instead of building the full table or the entire DAG. To change the default settings, you can select **No deferral** or choose a different environment. 
```

Comment:
You can also do self-deferral after the job has run. Do we need to mention that?